gulpfile.js

```diff
@@ -120,28 +126,34 @@ gulp.task('download-7zip-extra', function() {
 gulp.task('unzip-7zip-extra', function(cb) {
   let cmd = zaExe + ' e ' + zaExtra7z + ' -o' + zaRoot + ' -y ' + '7zS.sfx';
   // console.log(cmd);
-  exec(cmd, createExecCallback(cb, true));
-});
+  exec(cmd, createExecCallback(cb,true));
+});
 
 gulp.task('download-resource-hacker', function() {
   return request('http://www.angusj.com/resourcehacker/resource_hacker.zip')
       .pipe(fs.createWriteStream(rhZip));
-});
+});
 
 gulp.task('unzip-resource-hacker', function() {
   return gulp.src(rhZip)
       .pipe(unzip({ filter : function(entry){ return minimatch(entry.path, "**/ResourceHacker.*") } }))
       .pipe(gulp.dest(buildFolderRoot));
 });
 
-gulp.task('prepare-tools', function(cb) {
+gulp.task('prepare-tools', function(cb) {
   runSequence(['download-7zip', 'download-7zip-extra', 'download-resource-hacker'],
     ['unzip-7zip', 'unzip-resource-hacker'], 'unzip-7zip-extra', cb);
 });
 
 // wrap electron-generated app to 7zip archive
 gulp.task('create-7zip-archive', function(cb) {
-  let packCmd = zaExe + ' a ' + bundled7z + ' ' + zaElectronPackage + path.sep + '*';
+  var packCmd = zaExe + ' a ' + bundled7z + ' ' + zaElectronPackage + path.sep + '*'

```

Comment:
Can you be more specific, please? `let` doesn't influence variable's visibility inside `if` statement and we don't need to use the variable before it's declared.